common/src/main/java/io/github/gaming32/bingo/triggers/ChickenHatchTrigger.java

```diff
@@ -1,14 +1,11 @@
 package io.github.gaming32.bingo.triggers;
 
-import com.google.gson.JsonObject;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
 import net.minecraft.advancements.Criterion;
-import net.minecraft.advancements.critereon.AbstractCriterionTriggerInstance;
-import net.minecraft.advancements.critereon.ContextAwarePredicate;
-import net.minecraft.advancements.critereon.DeserializationContext;
-import net.minecraft.advancements.critereon.EntityPredicate;
-import net.minecraft.advancements.critereon.MinMaxBounds;
-import net.minecraft.advancements.critereon.SimpleCriterionTrigger;
+import net.minecraft.advancements.critereon.*;
```

Comment:
Wildcard import

Reply:
This matches IntelliJ's default wildcard import folding threshold

Reply:
Yeah which is bad and goes against pretty much every style guide I've seen. Increase the threshold to 9999 or something